pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
Good points, @vhaasteren . Plus, if we set arbitrary limits on array sizes by setting a variable called MAXDERIVS, we are in danger of looking like FORTRAN programmers. All we are missing is the all caps constant name.  :-)